daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DataProcessor.scala

```diff
@@ -145,7 +137,6 @@ class DataProcessor private (
   var validationMode: ValidationMode.Type,
   private var externalVars: Queue[Binding])
   extends DFDL.DataProcessor
-  with HasSetDebugger
```

Comment:
There's a comment just above that says:
```scala
  //
  // Once the deprecated API goes away, these var can become val.
  //
```
We should probably make that change now.

Reply:
This will be added in the next commit